betterttv.js

```diff
@@ -4098,7 +4078,449 @@ module.exports = function () {
     }
 }
-},{"../helpers/debug":43}],42:[function(require,module,exports){
+},{"../helpers/debug":47}],43:[function(require,module,exports){
+var debug = require("../../debug"),
+    vars = require("../../vars"),
+    tseInnerTemplate = require('../../templates/team_tse-content-inner-div'),
+    followButtonTemplate = require('../../templates/team_follow-button'),
+    loadTeam = require("./team-load-team");
+
+module.exports = function () {
+    if(bttv.settings.get("formatTeamPage") !== true || $("body").attr("data-page") != "teams#show") return;
+    
+    vars.teamCurrentChannel = ($(".js-playing").attr("id")).replace("channel_", "");
+    vars.teamFirstLoad = 1;
+    vars.teamName = window.location.pathname.replace(/\/team\/|\/event\//, "");
+    
+    var teamFollowList = "none",
+        teamFollowedFailedList = "none",
+        teamFollowRequestCount = 0,
+        teamFollowResponseCount = 0,
+        teamFollowInterval = "none";
+        
+    $(window).resize(function() {
+        var vpWidth = $(window).width(),
+            vpHeight = $(window).height(),
+            paddedWidth = vpWidth - 40,
+            paddedHeight = vpHeight - ($("#site_header").outerHeight(true) + 40),
+            teamListHeight = paddedHeight - ($("#team_logo").outerHeight(true) + $("#bttvFollowTeamButton").outerHeight(true) + 2),
+            playerWidth = paddedWidth - ($(".filters.grid.c4").outerWidth(true) + $("#bttvTeamChat").outerWidth(true) + 2),
+            arMultiplier = (playerWidth) / 16,
+            playerHeight = (9 * arMultiplier) + 32;
+            
+        if(paddedWidth < 985) {
+            paddedWidth = 985;
+        }
+
+        $("div.main.discovery.team").css("width", vpWidth+"px");
+        $("div.wrapper.c12.clearfix").css("width", paddedWidth+"px");
+        $("#player_column").css({"width":(playerWidth+2)+"px", "height":paddedHeight+"px"});
+        $("#site_footer").css("width", playerWidth+"px");
+        $("#standard_holder").css({"width":playerWidth+"px", "height":playerHeight+"px"});
+        $("#bttvTeamChat").css("height", (paddedHeight-2)+"px");
+        $("#bttvTeamMemberListOuter").css({"height":teamListHeight+"px"});
+    });
+    
+    var betterTeamInit = function() {
+        var teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+        
+        if(bttv.settings.get("darkenedMode") === true) {
+            teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page-dark.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+        }
+        $("body").append(teamCSS);
+        
+        getFeatured();
+        formatTeamPage();
+        //if it wasn't for the bttv.TwitchAPI._ref is null error in Firefox, I would just load the team without the timeout
+        setTimeout(loadTeam, 1000);
```

Comment:
Is there any reason why you inject the emote script on pages that do not have chat? IE: Current team pages, messages, etc? Because every time that happens, I notice the error (except that it's now: "e.TwitchAPI._ref is null", instead of "bttv.TwitchAPI_ref is null") in my console on Firefox, right after the "Injecting Twitch Chat Emotes Script" debug message. This also causes a huge Angular error (referencing bttvEmotes) to follow shortly after. I didn't include the error message because of long it is (but will upon request).

I would like to note, this happens even when I'm NOT using my local files for BTTV.